src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1311,6 +1369,27 @@ generator client {
             } else {
               acc[key] = value
             }
+            if (key === 'count') {
+              if (typeof value === 'object' && value) {
+                acc.select[key] = { select: mapAllCount(value) }
+                console.log(acc.select[key]);
+                unpacker = (data) => {
+                  console.log(data);
+                  if (data?.[0]?.count && typeof data[0].count === 'object') {
```

Comment:
Need some eyes here

Reply:
Instead, I suggest the following:
```ts
if (Array.isArray(data)) {
  data = data.map(row => {
    if (row.count) {
		row.count = mapAllCount(row.count)
	}
	return row
  })
}
```
We never know, maybe one row has it, one not. We can do this mutable, so it's faster.